debian/clearwater-socket-factory.postinst

```diff
@@ -54,8 +54,8 @@ set -e
 case "$1" in
     configure)
         # Start the upstart jobs
-        start clearwater-socket-factory-mgmt
-        start clearwater-socket-factory-sig
+        start clearwater-socket-factory-mgmt || true
```

Comment:
It is probably worth a comment explaining why we are ignoring the error (because the services may already be running).